### ISSUE TITLE
Biomes O' Plenty issue?

### ISSUE THREAD

**Issue by KnownOriginator** [AllTheMods/atm9nf#66]
Okay, so I originally thought it was something else, but I have successfully narrowed it down.
If you go into the config folder for the server and manually change all the biome toggles for Biomes O' Plenty to *false* like I did cause I just don't like the biomes. The game will then crash when you join the server and try to place a crafting table.

Okay, I'll just remove the Biomes O' Plenty mod entirely from the server! It launches and I can join, but now my game crashes when I try to just open my inventory.

Summing up: for people who want to run/play a server while disabling BOP biomes or the mod itself...well they just can't, it don't work.
Sadface.

Edit: I'm gonna be testing which biomes I can disable in the config, try to see which one specifically makes the game crash.
Edit part 2: Electric Boogaloo: If you disable "Bog" in the BOP biome_toggles.json file, the game will crash when you try to place a crafting table.

So I've disabled everything *but* bog and we'll see what happens!

**Comment by KnownOriginator** [comment]
And yes, I'm aware I'm likely the _only_ person with this issue, but I digress.
